core/server/master/src/main/java/alluxio/web/WebInterfaceBrowseServlet.java

```diff
@@ -135,158 +135,162 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     if (SecurityUtils.isSecurityEnabled() && AuthenticatedClientUser.get() == null) {
       AuthenticatedClientUser.set(LoginUser.get().getName());
     }
-    request.setAttribute("debug", Configuration.getBoolean(PropertyKey.DEBUG));
-    request.setAttribute("showPermissions",
-        Configuration.getBoolean(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED));
+    if (Configuration.getBoolean(PropertyKey.WEB_FILE_INFO_ENABLED)) {
+      // browse-body.jsp is used by many web pages and we only want to disable the browse page
```

Comment:
what page uses browse-body.jsp, but should still be shown when file info is disabled?

Reply:
The logs page also use the browse-body.jsp and do not need to be disabled. The downloads page also use it (which looks strange to me...)

Reply:
I think we need to disable the logs page as well since logs contain file information like file names. And users shouldn't be able to download data

Reply:
Changed to disable all the logs/downloads/browse/blockInfo pages, Thanks!